src/main/java/pl/javastart/task/PalindromeService.java

```diff
@@ -4,6 +4,15 @@ public class PalindromeService {
 
     // nie zmieniaj sygnatury tej metody. Jest ona testowana w PalindromeServiceTest
     public boolean isPalindrome(int[] array) {
-        return false;
+        if (array == null || array.length == 0) {
+            return false;
+        }
+
+        for (int i = 0; i < array.length; i++) {
```

Comment:
Jeżeli mamy tablicę np.
[1, 2, 3, 3, 2, 1]
to wystarczą nam 3 porównania, a nie 6 (liczbę iteracji pętli można zmniejszyć)